.github/workflows/release.yml

```diff
@@ -51,6 +51,8 @@ jobs:
         with:
           fetch-depth: 0
           submodules: recursive
+          fetch-tags: true
```

Comment:
I believe fetch-depth 0 should fetch the entire history. Also note, fetch-depth 0 fetch the origin main head, which is fine for release but I am not sure if it's the correct behaviour we want.

Reply:
Yeah, I wasn't sure why this was `0` in the first place, so I left it. It should be `1` here too. @nravic Was there a reason for this being `0`?

Reply:
Hmm no reason